tide_authenticated_content.install

```diff
@@ -47,6 +55,71 @@ function tide_authenticated_content_update_8002() {
  */
 function tide_authenticated_content_update_8003() {
   module_load_include('inc', 'tide_core', 'includes/helpers');
-  $config_location = [drupal_get_path('module', 'tide_authenticated_content') . '/config/install'];
+  $config_location = [
+    drupal_get_path('module',
+    'tide_authenticated_content') . '/config/install',
+  ];
   _tide_import_single_config('tide_authenticated_content.module', $config_location, TRUE);
 }
+
+/**
+ * Implements hook_uninstall().
+ */
+function tide_authenticated_content_uninstall() {
+  // Remove permissions on tide_authenticated_content to Approver and Editor.
+  $roles = ['approver', 'editor'];
+  $permissions = [
+    'create terms in authenticated_content',
+    'create terms in authenticated_content',
+  ];
+
+  foreach ($roles as $role_name) {
+    $role = Role::load($role_name);
+    if ($role) {
+      foreach ($permissions as $permission) {
+        $role->revokePermission($permission);
+      }
+      $role->save();
+    }
+  }
+  // Remove taxonomy and fields that tide_authenticated_content has installed.
+  Vocabulary::load('authenticated_content')->delete();
+
+  // Clean up paragraphs.
+  $paragraph_query = \Drupal::entityTypeManager()
+    ->getStorage('paragraph')
+    ->getQuery();
+  $results = $paragraph_query->condition('type', 'user_authentication_block')
+    ->execute();
+  if ($results) {
+    $paragraphs = Paragraph::loadMultiple($results);
+    \Drupal::entityTypeManager()->getStorage('paragraph')->delete($paragraphs);
+  }
+  $paragraph_type_entity = ParagraphsType::load('user_authentication_block');
+  \Drupal::entityTypeManager()
+    ->getStorage('paragraphs_type')
+    ->delete([$paragraph_type_entity]);
+  // Clean up nodes.
+  FieldConfig::loadByName('node', 'landing_page', 'field_authenticated_content')
+    ->delete();
+  FieldStorageConfig::loadByName('node', 'field_authenticated_content')
+    ->delete();
+  $node_types = NodeType::loadMultiple();
+  if ($node_types) {
+    foreach ($node_types as $node_type => $details) {
+      $node_fields = \Drupal::service('entity_field.manager')
+        ->getFieldDefinitions('node', $node_type);
+      if (isset($node_fields['field_authenticated_content'])) {
```

Comment:
Please verify $node_fields is an array.

Reply:
it is checking, the results will be an array as always.